test/dotnet-new3.UnitTests/Sdk/Assertions/FileInfoExtensions.FileInfoLock.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
Is there a reason this file in Assertions folder?

Reply:
Its Copy from upstream dotnet/sdk repo...